src/arch/armv8/cache.c

```diff
@@ -42,6 +42,9 @@ void cache_arch_enumerate(struct cache* dscrp)
                 case CLIDR_CTYPE_UN:
                     dscrp->type[i] = UNIFIED;
                     break;
+                default:
+                    dscrp->type[i] = UNIFIED;
```

Comment:
As we hadn't had a default case before, shouldn't this return an error / hang?  or at least a warning